addons/binding/org.openhab.binding.energenie/src/main/java/org/openhab/binding/energenie/internal/api/JsonResponseHandler.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.energenie.internal.api;
+
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.openhab.binding.energenie.internal.api.constants.JsonResponseConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonParser;
+
+/**
+ * Contains some common methods which are used for processing the server's
+ * response
+ *
+ * @author Mihaela Memova - Initial contribution
+ *
+ */
+public class JsonResponseHandler {
```

Comment:
Usually, a "handler" is a class instance - so having all methods static is a bit unusual. If there isn't any strong reason for it, better make it a proper class that can be instantiated.

Reply:
Alternatively rather rename it to "...Helper" or "...Util".